source/agora/node/Ledger.d

```diff
@@ -44,7 +46,15 @@ public class Ledger
     public this (TransactionPool pool)
     {
         this.pool = pool;
+
+        string path = buildPath(getcwd, ".cache");
+        if (!path.exists)
+            mkdir(path);
+        this.ledger = new BlockStorage(path);
+        BlockStorage.removeIndexFile(path);
```

Comment:
I don't understand why the index file is deleted after loading? Also `removeIndexFile ` should be private and called from within the `BlockStorage` constructor.

Reply:
deleted `removeIndexFile`

Reply:
However, we have to declare it separately during the unittest.